README.md

```diff
@@ -1129,6 +1129,7 @@ A GitHub conflict is when people make changes to the same area or line in a file
   - [Sayed Mohammed Nouman](https://github.com/Skyhero-admin)
   - [schmelto](https://github.com/schmelto)
   - [Sean Michael Motale](https://github.com/mainfunktion)
+  - [Sebastian Alexander](https://github.com/sex-lex)
```

Comment:
```suggestion
  - [Sebastian Alexander](https://github.com/seb-lex)
```

Reply:
Oh! Thank you for that, schmelto. Changes made.